src/sql/workbench/api/node/sqlExtHost.api.impl.ts

```diff
@@ -772,7 +772,7 @@ export function createApiFactory(
 				return extHostDataProvider.$registerAdminServicesProvider(provider);
 			};
 
-			let registerAgentServicesProvider = (provider: sqlops.AgentServicesProvider): vscode.Disposable => {
+			let registerAgentServicesProvider = (provider: azdata.AgentServicesProvider): vscode.Disposable => {
```

Comment:
we shouldn't reference the `azdata` module from the `sqlops` module since sqlops is for backcompat.  Since this is for agent which isn't used outside our code maybe we should just remove this provider from the sqlops module.

Reply:
I can add AgentService to the mssql extension and import it from there, and remove it from the sqlops module. Does that work?

Reply:
yeah, that would be better.  Let's get it out of the core extensibility in any case.

Reply:
@kburtram I'll put out another PR for removing agent out of core and putting it all into one extension. It requires some more work so, I'll let this PR be like this for now and get this merged before March release.

Reply:
why does the sqlops namespace need to be updated at all?  isn't the extension only using azdata now?

Reply:
yes, that's why i got rid of it from sqlops because we don't use it any more
i can let it remain unchanged for this PR